.eslintrc.js

```diff
@@ -87,6 +92,22 @@ module.exports = {
         'no-use-before-define': 'off'
       }
     },
+    {
+      files: [
+        'lib/**/*-after.js',
+        'lib/core/reporters/**/*.js'
+      ],
+      parserOptions: {
+        sourceType: 'module'
+      },
+      env: {},
```

Comment:
To avoid the exclusion above, just do:

```suggestion
      env: {
        browser: false,
        etc: ...
      },
```